extras/uwsgi/ideascube.conf

```diff
@@ -1,5 +0,0 @@
-[Service]
-User=ideascube
-Group=ideascube
-PAMName=ideascube
-Environment=SHELL=/bin/bash
```

Comment:
Good point ! The other web app are already using the uid and gid in their *.ini, so it should work as good as before